email/parser.go

```diff
@@ -93,6 +104,23 @@ func parseMessageWithHeader(headers Header, bodyReader io.Reader) (*Message, err
 	}, nil
 }
 
+// removeDuplicate removes duplicate from bogus content type preventing parsing of email
+func removeDuplicate(s string) string {
+	m := make(map[string]bool)
+	parts := strings.Split(s, ";")
+	for _, p := range parts {
+		if _, ok := m[p]; !ok {
+			m[p] = true
+		}
+	}
+
+	var cleaned string
+	for k := range m {
+		cleaned += k + ";"
```

Comment:
Nevermind, I got ahead of myself, it is a map not a slice/array.  I guess you could use a `bytes.Buffer`, though it won't really matter for a small number of entries.